client/scripts/controllers/chain/substrate/account.ts

```diff
@@ -112,12 +120,27 @@ class SubstrateAccounts implements IAccountsModule<SubstrateCoin, SubstrateAccou
         of(api),
         api.derive.staking.validators(),
         api.query.staking.currentEra(),
+        api.derive.staking.electedInfo()
       )),
 
       // fetch balances alongside validators
-      flatMap(([api, { nextElected, validators: currentSet }, era]: [ApiRx, DeriveStakingValidators, EraIndex]) => {
+      flatMap(([api, { nextElected, validators: currentSet }, era, electedInfo]: [ApiRx, DeriveStakingValidators, EraIndex, DeriveStakingElected]) => {
         // set of not yet but future validators
         const toBeElected = nextElected.filter((v) => !currentSet.includes(v));
+        const validatorsInfo: IValidatorInfo = {};
+
+        electedInfo.info.forEach(({ accountId, validatorPrefs }) => {
+          const commissionPer = (validatorPrefs.commission.unwrap() || new BN(0)).toNumber() / 10_000_000;
```

Comment:
Can you explain this line here? Why do we divide by 10,000,000?

Reply:
Oh I see, you're converting parts-per-billion to a percentage -- might be worth documenting that, but looks OK otherwise.

Reply:
Yap, you are right.